clang/test/CXX/dcl/dcl.init/aggr.cpp

```diff
@@ -0,0 +1,294 @@
+// RUN:  %clang_cc1 -std=c++2c -verify %s
+
```

Comment:
For future reference, can you say this is testing c++2c and maybe quote a paper number (latest working draft?)

Reply:
Any reference would become outdated the moment this file is touched next time, so I added one to the PR description, so it makes it into the commit message.